package.json

```diff
@@ -36,25 +36,25 @@
   },
   "homepage": "https://github.com/mapbox/mapbox-gl-geocoder#readme",
   "devDependencies": {
-    "brfs": "1.4.3",
-    "browserify": "^13.0.1",
-    "budo": "^9.2.1",
-    "documentation": "^4.0.0-beta13",
+    "brfs": "^2.0.2",
+    "browserify": "^16.2.3",
+    "budo": "^11.6.1",
+    "documentation": "^9.3.1",
```

Comment:
@scottsfarley93 since this is a pretty big version bump, we may want to run `npm run docs` in this branch to check to see if there are any output differences and commit any changes to API.md

Reply:
👍 Doesn't look to me like there is any difference in the output from `npm run docs`.